safe-global/safe-wallet-web
# feat: Use different titles depending on the type of swap

## What it solves

Resolves [Notion issue](https://www.notion.so/safe-global/Issue-No-icon-and-wrong-name-on-the-confirmation-screen-7d49cf4e40794dd4b9a628e192c24f43)

Note: This doesn't add titles for specific actions like "Wrap & Swap" and "Approve & Swap"

## How this PR fixes it

- Shows either `Swap order` or `Limit order` as the title when creating a swap tx via the swap widget
- Adjusts the `AppTitle` component to be consistent with the styles of other transaction titles and icons

## How to test it

1. Create a limit and a swap order
2. Observe the correct title showing
3. Queue them
4. Go to the queue and press sign/execute
5. Observe the correct title showing

## Screenshots

## Checklist
